src/index.ts

```diff
@@ -1,2 +1,9 @@
-export * from './vanilla'
-export * from './react'
+/* 
+  Dynamic re-exports blocked to support snowpack ts re-export issues
+*/
+// export * from "./vanilla"
+// export * from "./react"
+
+export { useSnapshot } from './react'
+
+export { ref, proxy, getVersion, subscribe, snapshot } from './vanilla'
```

Comment:
```suggestion
export { ref, proxy, getVersion, subscribe, snapshot } from './vanilla'
export { useSnapshot } from './react'
```

just a preference..

Reply:
actually,
```ts
export * from './react'
```
is not a problem?

Reply:
Yeah, surprisingly that works but I did it for consistency in the export
As for how I got to know that this fix works is when I manually added the named exports in the compiled ESM file
talking about 
`dist/esm/index.mjs` and it imported the contents of compiled `react.ts` but not for vanilla and since the rollup was aliasing vanilla I didn't want to remove it so I added the named exports manually to test and it worked. 

There is one another solution, if you want me to try that, I can do that on my fork and send through the links for the same

Reply:
I can understand the preference on the consistency.
I think the current workaround is good enough.